### PR TITLE
Removes Banhammer

### PR DESCRIPTION
**Pull request by ynot01**
## Intent of your Pull Request

OOC in IC
Some instances that spawned the banhammer now spawn a bike horn instead

#### Changelog

:cl:  
rscdel: Removed the banhammer
tweak: Some instances that spawned the banhammer now spawn a bike horn instead
/:cl:

### CODE REVIEW

**Comment by jcatjmeow** [comment]
Doesnt FUCKING COMPILE

But I do agree with this change

**Comment by ynot01** [comment]
> Doesnt FUCKING COMPILE
> 
> But I do agree with this change

FUDK yo

**Comment by maxion12345** [comment]
Fcuk you get baneNdx four no rasin

**Comment by athena413** [comment]
as funny as the banhammer is, yeah, it's OOC in IC

**Comment by patpol4** [comment]
Don't see a reason to remove it from the code, just erase all instances of it existing and keep it admin only

**Comment by ynot01** [comment]
> Don't see a reason to remove it from the code, just erase all instances of it existing and keep it admin only

> Dont add things that are ""bus only"", only ends in unmaintained bloat features

**Comment by boodaliboo** [comment]
add it back to code

**Comment by ynot01** [comment]
there is no reason for it to exist, it is pointless in any bus scenario and would still be OOC in IC in those bus scenarios

**Comment by Bibby0110** [comment]
Remove it from the map not the code

**Review by patpol4 — APPROVED** [review]
Up to headcoders then, it would be sad to see something go like this. Map is fine.

File: code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -1,36 +1,3 @@
...
-/obj/item/banhammer/attack(mob/M, mob/user)
-	if(user.zone_selected == BODY_ZONE_HEAD)
-		M.visible_message("<span class='danger'>[user] are stroking the head of [M] with a bangammer</span>", "<span class='userdanger'>[user] are stroking the head with a bangammer</span>", "you hear a bangammer stroking a head");
-	else
-		M.visible_message("<span class='danger'>[M] has been banned FOR NO REISIN by [user]</span>", "<span class='userdanger'>You have been banned FOR NO REISIN by [user]</span>", "you hear a banhammer banning someone")
-	playsound(loc, 'sound/effects/adminhelp.ogg', 15) //keep it at 15% volume so people don't jump out of their skin too much
-	if(user.a_intent != INTENT_HELP)
-		return ..(M, user)
-
 /obj/item/sord
```
Comment (JamieD1) [inline code/game/objects/items/weaponry.dm]:
```suggestion
/obj/item/banhammer
	desc = "A banhammer."
	name = "banhammer"
	icon = 'icons/obj/items_and_weapons.dmi'
	icon_state = "toyhammer"
	slot_flags = ITEM_SLOT_BELT
	throwforce = 0
	force = 1
	w_class = WEIGHT_CLASS_TINY
	throw_speed = 3
	throw_range = 7
	attack_verb = list("banned")
	max_integrity = 200
	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 100, "acid" = 70)
	resistance_flags = FIRE_PROOF

/obj/item/banhammer/suicide_act(mob/user)
		user.visible_message("<span class='suicide'>[user] is hitting [user.p_them()]self with [src]! It looks like [user.p_theyre()] trying to ban [user.p_them()]self from life.</span>")
		return (BRUTELOSS|FIRELOSS|TOXLOSS|OXYLOSS)
/* 
oranges says: This is a meme relating to the english translation of the ss13 russian wiki page on lurkmore.
mrdoombringer sez: and remember kids, if you try and PR a fix for this item's grammar, you are admitting that you are, indeed, a newfriend.
for further reading, please see: https://github.com/tgstation/tgstation/pull/30173 and https://translate.google.com/translate?sl=auto&tl=en&js=y&prev=_t&hl=en&ie=UTF-8&u=%2F%2Flurkmore.to%2FSS13&edit-text=&act=url
*/
/obj/item/banhammer/attack(mob/M, mob/user)
	if(user.zone_selected == BODY_ZONE_HEAD)
		M.visible_message("<span class='danger'>[user] are stroking the head of [M] with a bangammer</span>", "<span class='userdanger'>[user] are stroking the head with a bangammer</span>", "you hear a bangammer stroking a head");
	else
		M.visible_message("<span class='danger'>[M] has been banned FOR NO REISIN by [user]</span>", "<span class='userdanger'>You have been banned FOR NO REISIN by [user]</span>", "you hear a banhammer banning someone")
	playsound(loc, 'sound/effects/adminhelp.ogg', 15) //keep it at 15% volume so people don't jump out of their skin too much
	if(user.a_intent != INTENT_HELP)
		return ..(M, user)

/obj/item/sord
```

**Review by JamieD1 — CHANGES_REQUESTED** [review]

**Pull request merged**